documentation/lldb/buildinglldb.md

```diff
@@ -1,6 +1,8 @@
 Building LLDB
```

Comment:
Lets add a note saying this is legacy and pointing most people back to the main directions